app/black_spots/models.py

```diff
@@ -0,0 +1,38 @@
+import uuid as uuidlib
+
+from django.conf import settings
+from django.contrib.gis.db import models
+
+from ashlar.models import AshlarModel
+
+
+class BlackSpot(AshlarModel):
+    """A black spot -- an area where there is an historical/statistical concentration of records"""
+
+    #: Buffered road segment polygon where black spot analysis is performed
+    geom = models.PolygonField(srid=settings.ASHLAR['SRID'])
+
+    #: Number that determines the severity of this black spot. May be used for color-coding on map.
+    severity_score = models.FloatField()
+
+    #: Number of records accounted for in the polygon while analyzing
+    num_records = models.PositiveIntegerField()
```

Comment:
I made them fields because they represent the state of the data at the point in time in which the future machine-learning algorithm calculates and inserts the black spots into the db. It'll have that information at the time, and that information may possibly change in the future if records are deleted or edited (i.e. Department of Health updates records to fatal). So I think it makes sense to store the data in the db to have a snapshot of what the polygon looked like at the time of processing.